libs/ui-lib-tests/cypress/integration/create-multinode/2-networking.cy.ts

```diff
@@ -1,36 +1,37 @@
 import { commonActions } from '../../views/common';
 import { networkingPage } from '../../views/networkingPage';
-import * as utils from '../../support/utils';
-import { NetworkingRequest } from '../../fixtures/create-mn/requests';
+import { getNetworkingRequest } from '../../fixtures/create-mn/requests';
 
 describe(`Assisted Installer Multinode Networking`, () => {
   before(() => {
     cy.loadAiAPIIntercepts({
-      activeSignal: 'HOST_RENAMED_1',
+      activeSignal: 'READY_TO_INSTALL',
```

Comment:
I spent a lot of time in my PR (https://github.com/openshift-assisted/assisted-installer-ui/pull/2233) making changes to the validations info fixtures to make the mocks closer resemble the actual life-cycle of a cluster. This feels like a step back.

When the user first accesses the Networking page, the cluster doesn't have `machineNetworks`, `apiVips`, and `ingressVips` defined. These values are not provided automatically by the BE, the users need to enter them manually. Until then, there are a lot of pending validations that prevent the user from continuing past the Networking page.

Maybe we could do something like this here:

```

describe(`Assisted Installer Multinode Networking`, () => {
  describe('Before finishing networking configuration', () => {
    before(() => {
      cy.loadAiAPIIntercepts({
        activeSignal: 'HOST_RENAMED_3',
        activeScenario: 'AI_CREATE_MULTINODE',
      });
    });

    beforeEach(() => {
      cy.loadAiAPIIntercepts(null);
      commonActions.visitClusterDetailsPage();
      // because of the failing networking validations, a cluster at this stage will start on the Host discovery page
      commonActions.moveNextSteps(['Host discovery', 'Storage']);
    });

    it('Should submit updated network information', () => {
     ...
    });
  });

  describe('After finishing networking configuration', () => {
    before(() => {
      cy.loadAiAPIIntercepts({
        activeSignal: 'READY_TO_INSTALL',
        activeScenario: 'AI_CREATE_MULTINODE',
      });
    });

    beforeEach(() => {
      cy.loadAiAPIIntercepts(null);
      commonActions.visitClusterDetailsPage();
      commonActions.startAtWizardStep('Networking');
    });

    it('Should see the Ready Host inventory status', () => {
     ...
    });
    ...
  });
});
```

Reply:
You are right, my changes did change the approach.
I have updated the tests with two describe as you suggest, and added some additional assertions to make sure we're testing what the tests say. PTAL.

Reply:
LGTM